test-suite/Makefile

```diff
@@ -657,6 +657,7 @@ coq-makefile/%.log : coq-makefile/%/run.sh
 	@echo "TEST      coq-makefile/$*"
 	$(HIDE)(\
 	  export COQBIN=$(BIN);\
+	  export COQLIB=$(LIB)/../../;\
```

Comment:
???
Maybe you want to use the absolute value for LIB?

Reply:
There is no and easy way to generate an absolute path that will work well for Windows.

Reply:
We had this discussion in the `-boot` PR and you seemed to favor relative paths, why the change?

Reply:
The issue is that here we're `cd`ing so the relative gets invalidated
But LIB isn't always relative, see gitlab.yml, so unconditionally adding ../.. doesn't work.